.classpath

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In the future projects this file should be on the .gitignore list (we've done that in the AH5)